src/geodeapp_back/app.py

```diff
@@ -0,0 +1,119 @@
+""" Packages """
+
+import os
+import dotenv
+
+import flask
+import flask_cors
+import time
+
+
+from opengeodeweb_back import geode_functions
+from opengeodeweb_back.routes import blueprint_routes
+
+from werkzeug.exceptions import HTTPException
+
+from werkzeug.exceptions import HTTPException
+
+from .config import *
+
+
+if os.path.isfile("./.env"):
+    basedir = os.path.abspath(os.path.dirname(__file__))
+    dotenv.load_dotenv(os.path.join(basedir, ".env"))
+
+""" Global config """
+app = flask.Flask(__name__)
+
+
+def kill_task():
```

Comment:
this function and the exception handler could be in OGW-Back?

Reply:
yes